pg/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rubybench/ruby:0.5
+
+LABEL maintainer="https://github.com/bmarkons"
+
+ADD runner runner
+
+RUN chmod 755 runner
+RUN apt-get update && apt-get install -y \
+      rubygems \
```

Comment:
it works fine when called with `bin/bash` like here :
https://github.com/ruby-bench/ruby-bench-docker/blob/master/rails/rails_trunk/rails_benchmarks/Dockerfile#L10

Though I played around and figured out it can be called with `bin/sh` if you install `rubygems` explicitly.

I can leave it as it is, or switch to bin/bash. Either way works fine.